server/routes/amendAReferral/changelog/changelogPresenter.ts

```diff
@@ -22,6 +25,10 @@ export default class ChangelogPresenter {
 
   readonly errorSummary = PresenterUtils.errorSummary(this.error)
 
+  readonly text = {
+    title: `${utils.convertToTitleCase(this.intervention.contractType.name)}: change log`,
```

Comment:
_not a blocker for the changes_

`contractType.name` is populated by us. What problem does `convertToTitleCase` solve?

I'm concerned because we have
```
interventions=# select name from contract_type;
                         name
------------------------------------------------------
 Accommodation
 Education, Training and Employment (ETE)
 Personal Wellbeing
 Women's Services
 Mentoring
 Dependency and Recovery
 Women's Support Services (GM)
 Finance, Benefit and Debt (East Midlands)
 Finance, Benefit and Debt (West Midlands)
 Finance, Benefit and Debt (London)
 Finance, Benefit and Debt (South Central)
 Dependency and Recovery (Yorkshire and the Humber)
 Dependency and Recovery (London)
 Dependency and Recovery (North East)
 Dependency and Recovery (Kent, Surrey and Sussex)
 Finance, Benefit and Debt in Custody (North East)
 Dependency and Recovery (North East: Northumbria)
 Finance, Benefit and Debt (Wales)
 Dependency and Recovery (South Central)
 Dependency and Recovery (South Yorkshire)
 Dependency and Recovery (West Midlands)
 Finance, Benefit and Debt (Yorkshire and the Humber)
 Finance, Benefit and Debt (North West)
(23 rows)
```

Reply:
For example, the education title will become `Education, Training And Employment (ete)`, which does not look very good

Reply:
@sldblog This is been already implemented in other tabs, I just implemented in this tab. Do you think we need to reconsider those tabs as well

Reply:
Not in this PR, but I'm sure the lowercase `(ete)` isn't what the original author intended

Reply:
Sure @sldblog , will take it up with Helen regarding this